lib/components/artist/shows.js

```diff
@@ -7,29 +7,101 @@ const { View, Text, StyleSheet, Image, ListView } = React;
 import Headline from '../text/headline';
 import SerifText from '../text/serif'
 import Show from './show'
+import OpaqueImageView from '../opaque_image_view'
+
+var url = 'https://www.pangeareptile.com/Images/care/baby.jpg'
 
-export default class Shows extends React.Component {
 
-  updateShowData(props) {
-    // this.setState({
-    //   dataSource: this.state.dataSource.cloneWithRows(this.props.show),
-    //   loaded: true,
-    // });
+export default class Shows extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentShowsDataSource: new ListView.DataSource({
+        rowHasChanged:(row1, row2) => row1 !== row2,
+      }),
+      pastShowsDataSource: new ListView.DataSource({
+        rowHasChanged:(row1, row2) => row1 != row2,
+      }),
+      loaded: false,
+    };
   }
 
-  renderShow() {
-    return <ShowView show={this.props.show} />
+  componentDidMount() {
+    this.setState({
+      currentShowsDataSource: this.state.currentShowsDataSource.cloneWithRows(this.props.shows),
+      pastShowsDataSource: this.state.pastShowsDataSource.cloneWithRows(this.props.shows),
+      loaded: true,
+    });
   }
 
   render() {
     return (
-      <Show show={this.props.show} />
+      <View style={styles.container}>
+        <SerifText style={styles.title}> Current & Upcoming Shows </SerifText>
+
+        <ListView
+          dataSource={this.state.currentShowsDataSource}
+          renderRow={this.renderShow}
+          style={styles.listView}
+        />
+
+        <View style={{height: 1, backgroundColor: 'grey', flex: 3, marginBottom: 20, paddingLeft: 0}}></View>
+
+        <SerifText style={styles.title}> Past Shows </SerifText>
+
+        <ListView
+          dataSource={this.state.pastShowsDataSource}
+          renderRow={this.renderPastShow}
+          style={styles.listView}
+          renderSeparator={(sectionID, rowID) => <View key={`${sectionID}-${rowID}`} style={styles.separator} />}
+         />
+
+      </View>
+    );
+  }
+
+  renderShow(show) {
+    return (
+      <Show show={show}/>
+    );
+  }
+
+  renderPastShow(show) {
+    return (
+      <View style={styles.pastShowContainer}>
+        <OpaqueImageView imageURL={url} style={{width: 75, height: 75}} aspectRatio={1}/>
```

Comment:
Still not loading images even with the right spelling. I'm wondering if maybe it's a networking issue? I couldn't load images with the RN ```Image``` component either.